client/lib/plans/features-list.js

```diff
@@ -1073,14 +1080,20 @@ export const FEATURES_LIST = {
 
 	[ constants.FEATURE_BACKUP_DAILY_V2 ]: {
 		getSlug: () => constants.FEATURE_BACKUP_DAILY_V2,
-		getTitle: () => i18n.translate( 'Automated daily site backups' ),
+		getTitle: () =>
+			POPULAR_PRODUCTS_OFFERING_VARIANT === abtest( POPULAR_PRODUCTS_OFFERING_AB_TEST )
+				? i18n.translate( 'Automated daily backups (off-site)' )
+				: i18n.translate( 'Automated daily site backups' ),
 	},
 
 	[ constants.FEATURE_BACKUP_REALTIME_V2 ]: {
 		getSlug: () => constants.FEATURE_BACKUP_REALTIME_V2,
 		getTitle: () =>
 			( {
-				i5: i18n.translate( 'Automated real-time backups' ),
+				i5:
+					POPULAR_PRODUCTS_OFFERING_VARIANT === abtest( POPULAR_PRODUCTS_OFFERING_AB_TEST )
+						? i18n.translate( 'Backup (real-time, off-site)' )
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Backup Real-time (off-site)' )` *ES Score: 11*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/pricing-page-ab-test)